### PR TITLE
feat(mui5):introducing isFilterCaseInSensitive prop for dual-list-select filterOption

### PR DESCRIPTION
**Pull request by AmirSaif**
**Description**

Please include a summary of the change.
introducing isFilterCaseInSensitive prop for dual-list-select filterOption.
isFilterCaseInSensitive : false  -> Filtering would be Case sensitive, CaTs would not match with cats.
isFilterCaseInSensitive : true -> Filtering would not be Case sensitive, CaTs would match with cats.
**Schema** *(if applicable)*
export default {
    "fields": [
      {
        "component": "customDualList",
        "name": "MUI",
        "isFilterCaseInSensitive":true,
        "options": [
          {
            "value": "cats",
            "label": "cats"
          },
          {
            "value": "cats_1",
            "label": "cats_1"
          },
          {
            "value": "cats_2",
            "label": "cats_2"
          },
          {
            "value": "zebras",
            "label": "zebras"
          },
          {
            "value": "pigeons",
            "label": "pigeons"
          }
        ]
      }
    ]
  };
```jsx
```

**Checklist:** *(please see [documentation page](https://data-driven-forms.org/development-setup) for more information)*

- [x] `Yarn build` passes
- [x] `Yarn lint` passes
- [x] `Yarn test` passes
- [x] Test coverage for new code *(if applicable)*
- [ ] Documentation update *(if applicable)*
- [x] Correct commit message
   - format `fix|feat({scope}): {description}`
   - i.e. `fix(pf3): wizard correctly handles next button`
   - fix will release a new \_.\_.X version
   - feat will release a new \_.X.\_ version (use when you introduce new features)
     - we want to avoid any breaking changes, please contact us, if there is no way how to avoid them
   - scope: package
   - if you update the documentation or tests, do not use this format
     - i.e. `Fix button on documenation example page`

### CODE REVIEW

File: packages/blueprint-component-mapper/src/dual-list-select/dual-list-select.d.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ interface InternalDualListSelectProps {
   filterOptionsText?: ReactNode;
   leftValues?: DualListSelectValue[];
   rightValues?: DualListSelectValue[];
+  isFilterCaseInSensitive?: boolean;
```
Comment (rvsia) [inline packages/blueprint-component-mapper/src/dual-list-select/dual-list-select.d.ts:29]: I would suggest to not use prop for it and set it as a default behavior. 

@Hyperkid123 what do you think?

Comment (Hyperkid123) [reply packages/blueprint-component-mapper/src/dual-list-select/dual-list-select.d.ts:29]: I'd go for default as well.

Comment (AmirSaif) [reply packages/blueprint-component-mapper/src/dual-list-select/dual-list-select.d.ts:29]: Raising new PR: https://github.com/data-driven-forms/react-forms/pull/1473, closing this one.

**Comment by AmirSaif** [comment]
Raised a new one: https://github.com/data-driven-forms/react-forms/pull/1473

**Pull request closed without merging**